packages/neon/neon/lib/src/pages/settings.dart

```diff
@@ -29,9 +30,12 @@ import 'package:provider/provider.dart';
 
 class SettingsPage extends StatefulWidget {
   const SettingsPage({
+    this.category,
     super.key,
   });
 
+  final String? category;
```

Comment:
Better to also name this initialCategory, otherwise the name is not really descriptive